mmv1/products/memorystore/Instance.yaml

```diff
@@ -0,0 +1,341 @@
+  encoder: templates/terraform/encoders/memorystore_instance.go.erb
+  decoder: templates/terraform/decoders/memorystore_instance.go.erb
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "memorystore_instance_basic"
+    primary_resource_id: "instance-basic"
+    min_version: beta
+    vars:
+      instance_name: "basic-instance"
+      policy_name: "my-policy"
+      subnet_name: "my-subnet"
+      network_name: "my-network"
+      prevent_destroy: "true"
+    test_vars_overrides:
+      prevent_destroy: "false"
+    oics_vars_overrides:
+      prevent_destroy: "false"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "memorystore_instance_full"
+    primary_resource_id: "instance-full"
+    min_version: beta
+    vars:
+      instance_name: "full-instance"
+      policy_name: "my-policy"
+      subnet_name: "my-subnet"
+      network_name: "my-network"
+      prevent_destroy: "true"
+    test_vars_overrides:
+      prevent_destroy: "false"
+    oics_vars_overrides:
+      prevent_destroy: "false"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "memorystore_instance_persistence_aof"
+    primary_resource_id: "instance-persistence-aof"
+    min_version: beta
+    vars:
+      instance_name: "aof-instance"
+      policy_name: "my-policy"
+      subnet_name: "my-subnet"
+      network_name: "my-network"
+      prevent_destroy: "true"
+    test_vars_overrides:
+      prevent_destroy: "false"
+    oics_vars_overrides:
+      prevent_destroy: "false"
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: "Identifier. Unique name of the instance.\nFormat: projects/{project}/locations/{location}/instances/{instance} "
+    output: true
+  - !ruby/object:Api::Type::String
+    name: createTime
+    description: "Output only. Creation timestamp of the instance. "
+    output: true
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    description: "Output only. Latest update timestamp of the instance. "
+    output: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: labels
+    description: "Optional. Labels to represent user-provided metadata. "
+  - !ruby/object:Api::Type::String
+    name: state
+    description:
+      "Output only. Current state of the instance. \n Possible values:\n
+      STATE_UNSPECIFIED\nCREATING\nACTIVE\nUPDATING\nDELETING"
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: targetShardCount
+            description: "Output only. Target number of shards for the instance. "
+            output: true
+          - !ruby/object:Api::Type::Integer
+            name: targetReplicaCount
+            description:
+              "Output only. Target number of replica nodes per shard for the
+              instance. "
+            output: true
+        name: updateInfo
```

Comment:
You probably want these parent fields to have `output: true` as well. Otherwise, it will look like the user can set them.

Reply:
Makes sense. Added.